server.cfg

```
@@ -22,77 +26,89 @@ sets sv_projectDesc "{{recipeDescription}}"
+add_ace group.admin command.quit deny # but don't allow quit
+add_ace resource.es_extended command.add_ace allow
+add_ace resource.es_extended command.add_principal allow
+add_ace resource.es_extended command.remove_principal allow
+add_ace resource.es_extended command.stop allow
+{{addPrincipalsMaster}}
+
+## EasyAdmin stuff
+setr ea_LanguageName "en"                # set our language to english
+setr ea_alwaysShowButtons "false"	# we don't want to see buttons we can't even access, do we? default: "false"
+set ea_moderationNotification "false"   # this can be either "false" or a discord webhook address, this will send a discord message if an admin takes actions against a player  ( such as banning and kicking )
+set ea_custombanlist "false"            # read docs for this, dont touch it
+set ea_enableCallAdminCommand "true" # enables "calladmin" command via webhook
+set ea_enableReportCommand "true" # enables "report" command, see Configuration Wiki Page for More infos
+setr ea_MenuButton "170"			# set our menu button to F3. default: "170"
+add_ace group.admin easyadmin allow
+
+
+
 ## These resources will start by default.
 ensure mapmanager
 ensure chat
 ensure spawnmanager
 ensure sessionmanager
-ensure hardcap
-ensure default_spawnpoint
-
-## Add system admins
-add_ace group.admin command allow # allow all commands
-add_ace group.admin command.quit deny # but don't allow quit
-{{addPrincipalsMaster}}
 
-## ESX v1-final Stuff
+## ESX Legacy
 ensure mysql-async
 ensure cron
-ensure instance
 ensure skinchanger
 ensure es_extended
 ensure esx_menu_default
 ensure esx_menu_dialog
 ensure esx_menu_list
-ensure esx_datastore
 ensure esx_identity
 ensure esx_skin
-ensure esx_status
-ensure esx_basicneeds
-ensure esx_billing
-ensure esx_addoninventory
-ensure esx_addonaccount
-ensure esx_society
-ensure esx_license
+
+## ESX Addons
+ensure esx_rpchat
 ensure esx_service
-ensure esx_garage
+ensure esx_shops
+ensure esx_sit
+ensure esx_society
+ensure esx_status
+ensure esx_taxijob
 ensure esx_vehicleshop
-ensure esx_jobs
-ensure esx_joblisting
-ensure esx_policejob
+ensure esx_voice
+ensure esx_weaponshop
+ensure esx_whitelist
+ensure instance
+ensure esx_accessories
+ensure esx_addonaccount
+ensure esx_addoninventory
 ensure esx_ambulancejob
-ensure esx_mechanicjob
-ensure esx_taxijob
-ensure esx_lscustom
+ensure esx_animations
 ensure esx_atm
-ensure esx_shops
-ensure esx_weaponshop
-ensure esx_clotheshop
+ensure esx_bankerjob
 ensure esx_barbershop
+ensure esx_basicneeds
+ensure esx_billing
+ensure esx_boat
+ensure esx_clotheshop
+ensure esx_cruisecontrol
+ensure esx_datastore
+ensure esx_dmvschool
 ensure esx_drugs
+ensure esx_garage
 ensure esx_holdup
+ensure esx_joblisting
+ensure esx_jobs
+ensure esx_license
+ensure esx_lscustom
+ensure esx_mechanicjob
+ensure esx_multicharacter
+ensure esx_optionalneeds
+ensure esx_phone
+ensure esx_policejob
+ensure esx_property
+ensure esx_realestateagentjob
+ensure 
```

This extra ensure is not required.
```suggestion
ensure 
```
Lines 74, 87 and 97 is not needed either, it just becomes a huge gap, like this:

`ensure esx_realestateagentjob
ensure



ensure EasyAdmin
`

I suggest it being shortened to this:
`ensure esx_realestateagentjob
ensure EasyAdmin
`

---

Hope this helps with this mate :)